test/src/org/labkey/test/tests/cds/CDSMAbTest.java

```diff
@@ -986,6 +989,7 @@ public void verifyMabAndSubjectGroups()
         Assert.assertTrue(SPECIES_COL + " should have been filtered", grid.isColumnFiltered(SPECIES_COL));
         Assert.assertFalse(MAB_COL + " should not have been filtered", grid.isColumnFiltered(MAB_COL));
         Assert.assertFalse(STUDIES_COL + " should not have been filtered", grid.isColumnFiltered(STUDIES_COL));
+*/
```

Comment:
Does it make sense to delete them instead of commenting it?

Reply:
sure I can delete it